utils/run.js

```diff
@@ -99,7 +101,12 @@ function runScript(pkgName, script) {
     }
     if (pkgJson && pkgJson.scripts && pkgJson.scripts[script]) {
       console.log(`\n[${pkgName}] Running yarn ${script}`);
-      const child = spawn('yarn', [script], { cwd, stdio: 'inherit' });
+      const child = spawn('yarn', [script], {
+        cwd,
+        stdio: 'inherit',
+        env: process.env,
```

Comment:
I don't think this one is necessary? 🤔

Reply:
This is necessary if you want to set `export FORCE_BUILD_IN_REGION=iad1` and run from the root. Its probably not necessary for this PR though 🤷‍♂

Reply:
It's strange because `process.env` is already the default value for `env` option: https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options/

> `env` _&lt;Object&gt;_ Environment key-value pairs. **Default**: `process.env`.

Reply:
Oh you're right! I reverted that line 👍